source/scripts/runAndroidTests.sh

```diff
@@ -13,6 +13,33 @@ UNITY_PATH=$(awk -F= '/^editor/ {print $2}' $scriptsDir/unity.properties)
 RESULTS_PATH="artifacts/android-test-results.xml"
 LOG_PATH="artifacts/androidTestsRun.log"
 
+# Check for any Android emulator / device
+connected_devices=$(adb devices | grep -v "List" | grep "device$")
+if [ -z "$connected_devices" ]; then
+    echo "No Android device connected."
+    
+    # Search for device name
+    DEVICE=$(emulator -list-avds | head -n 1)
+
+    echo "Launching $DEVICE, please wait a few seconds..."
+    ~/Library/Android/sdk/emulator/emulator "@$DEVICE" &
```

Comment:
Can you make it a property in `unity.properties`?

Reply:
could you please explain a bit more what you mean?